GliaWidgets/ViewModel/Chat/ChatViewModel.swift

```diff
@@ -42,6 +42,7 @@ class ChatViewModel: EngagementViewModel, ViewModel {
         )
         case showCallBubble(imageUrl: String?)
         case updateUnreadMessageIndicator(itemCount: Int)
+        case setOperatorTypingIndicatorVisibilityTo(Bool, _ isChatScrolledToBottom: Bool)
```

Comment:
It is not critical, just would like to propose to make the case name more simple and shorter. Probably something like this `setOperatorTypingIndicatorVisibilityTo` - > `operatorTypingIndiatorIsHidden`. Will we lose some context for this enumeration case?

Reply:
The only issue with your suggestion is that it doesn't convey an action in its name. With the `Action` enum cases we try to start them with an imperative verb. I don't like the name either, but I couldn't come up with a better one. 

With the current one the issue is also that we later do `indicator.isHidden = visibility` which is illogical. Passing `true` to that will hide the view.

Taking this into consideration, what about `setOperatorTypingIndicatorIsHiddenTo`? At least that makes more sense in terms of what true/false means.